azext_edge/edge/providers/check/common.py

```diff
@@ -326,6 +326,9 @@ class DataLakeConnectorTargetType(ListableEnum):
 # Lnm runtime attributes
 AIO_LNM_PREFIX = "aio-lnm"
```

Comment:
LNM stuff removed here :) https://github.com/Azure/azure-iot-ops-cli-extension/pull/246

Reply:
yeah let me merge the other one first, thanks!